docs/how-to-guides/adding-a-new-offline-store.md

```diff
@@ -6,9 +6,9 @@ Feast makes adding support for a new offline store (database) easy. Developers c
 
 In this guide, we will show you how to extend the existing File offline store and use in a feature repo. While we will be implementing a specific store, this guide should be representative for adding support for any new offline store.
 
-The full working code for this guide can be found at [feast-dev/feast-custom-offline-store-demo](https://github.com/feast-dev/feast-custom-offline-store-demo).
+The full working code for this guide can be found at [feast-dev/feast-custom-offline-store-demo](https://github.com/feast-dev/feast-custom-offline-store-demo) and an example of a custom offline store that was contributed by developers can be found [here](https://github.com/feast-dev/feast/pull/2401).
```

Comment:
idk if I would use this as an example. It's a really large PR with also online store + registry store components

Would be better if we created some example PR (similar to what is in this guide)

Reply:
Could you convert the dummy example in the guide into an actual PR so they can see? 

You can also just point users to the directory that has the offline / online store implementations too